src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -64,6 +63,8 @@ const Conversation: React.FC<ConversationProps> = props => {
     boolean
   >(false)
 
+  const [createsOfferOrder, setCreateOfferOrder] = useState<boolean>(true)
```

Comment:
Just a minor typo on the set:
```suggestion
  const [createsOfferOrder, setCreatesOfferOrder] = useState<boolean>(true)
```